src/dpsvalidationservice/src/main/resources/application.properties

```diff
@@ -3,3 +3,8 @@ server.port=8082
 # Swagger Ui
 dpsvalidation.service.api.version=0.0.1
 dpsvalidation.service.swagger.enabled=true
+
+# ORDS DFCMS client configuration
+dpsvalidation.service.ords.dfcms.client.base-url=
```

Comment:
Do the ORDS specific name value pairs not have to be populated to run this locally against ORDS?

Reply:
Yes it will need one to run locally, however we don't want to expose those routes in github, let's talk about how we can solve this. I really like the approach here to have something always configured by default but this one is tricky